event.c

```diff
@@ -399,17 +399,9 @@ event_handle_configurenotify(xcb_configure_notify_event_t *ev)
 {
     xcb_screen_t *screen = globalconf.screen;
 
-    if(ev->window == screen->root
-       && (ev->width != screen->width_in_pixels
-           || ev->height != screen->height_in_pixels))
+    if(ev->window == screen->root)
         /* it's not that we panic, but restart */
         awesome_restart();
-
-    /* Copy what XRRUpdateConfiguration() would do: Update the configuration */
-    if(ev->window == screen->root) {
-        screen->width_in_pixels = ev->width;
-        screen->height_in_pixels = ev->height;
-    }
```

Comment:
Sorry, but this reverts cb46a1163e904d37deee0e069cc81320ff34ea37 from just four days ago. The goal is to get rid of the `awesome_restart()` and handle screen changes without restarting. We are not there yet, which is why we still restart. We are on the way there, which is why we have this dead code.

Could you leave that part of the code intact?

(Also: Thanks for figuring out what kind of notify is sent for monitor changes! I don't think this is documented in the randr protocol docs.)